### DISCUSSION TITLE
User's perspective

### DISCUSSION THREAD

**Discussion by davidlv2** [Eloston/ungoogled-chromium#2760] [General]
From a users's perspective, when accessing the main Ungoogled Chromium site <https://github.com/ungoogled-software/ungoogled-chromium-macos/releases/> there is no way to easily see or comprehend the state of development. For example, when Cubix has started his build process, a user can see that the build in in progress in the Actions sections, but before that process is initiated, there is no way to tell what is the current state of affairs, i.e., when can the user expect a new build.
The latest Chromium version, to date, should be 122.0.6261.128, but that build has not appeared and the process for the next version Update to Chromium 123.0.6312.46 has already been initiated. Thus it may be good to have a "What's new" category in the menu system.

**Comment by networkException** [comment]
> the main Ungoogled Chromium site

Thats not the main site? Like, thats just the release page for macOS.

You can see releases and development in this repository (https://github.com/ungoogled-software/ungoogled-chromium)

If there's no pull request in the macOS repository Cubik has not started to work on packaging the release for macOS, anyone is free to do a bump themselves and open a pull request though

There's no guarantee that macOS is maintained. Maintainers are doing this work in their free time, on a best effort basis and with no responsibility to work on new releases unless they want to. I don't really know what you expect here.
